sos/report/plugins/ceph_mgr.py

```diff
@@ -103,17 +72,66 @@ def setup(self):
             "version"
         ]
 
+        directory = ''
+        if not microceph_pkg:
+            directory = '/var/run/ceph'
+            self.add_file_tags({
+                '/var/log/ceph/(.*/)?ceph-mgr.*.log': 'ceph_mgr_log',
+            })
+
+            self.add_forbidden_path([
+                "/etc/ceph/*keyring*",
+                "/var/lib/ceph/**/*keyring*",
+                "/var/lib/ceph/**/osd*",
+                "/var/lib/ceph/**/mon*",
+                # Excludes temporary ceph-osd mount location like
+                # /var/lib/ceph/tmp/mnt.XXXX from sos collection.
+                "/var/lib/ceph/**/tmp/*mnt*",
+                "/etc/ceph/*bindpass*",
+            ])
+
+            self.add_copy_spec([
+                "/var/log/ceph/**/ceph-mgr*.log",
+                "/var/lib/ceph/**/mgr*",
+                "/var/lib/ceph/**/bootstrap-mgr/",
+                "/run/ceph/**/ceph-mgr*",
+            ])
+
+        else:
+            directory = '/var/snap/microceph'
+            self.add_file_tags({
+                '/var/log/ceph/common/logs/ceph-mgr.*.log': 'ceph_mgr_log',
+            })
+
+            self.add_forbidden_path([
+                "/var/snap/microceph/common/**/*keyring*",
```

Comment:
minor nit, what about now using the directory variable too?

```python
f"{directory}/common/**/*keyring*",
```

Reply:
Not doing this for same reasons mentioned below, because we have some but not all common files

Reply:
I did manage to make the common add_cmd_output call though, thanks @arif-ali